### DISCUSSION TITLE
Is there any chance to simplify types?

### DISCUSSION THREAD

**Discussion by chiralium** [pmndrs/zustand#2222] [Q&A] [answered]
https://github.com/pmndrs/zustand/blob/50a8677dba4acecc1e313dcb08d4cf2ac1ab918c/src/middleware/devtools.ts#L39

**Comment by dai-shi** [comment, marked as answer]
Zustand types are trying to be pretty precise. But, that's not always easy for devs. Maybe there can be an escape hatch.
